portal/templates/utilisation-report-service/utilisation-report-upload/check-the-report.njk

```diff
@@ -0,0 +1,102 @@
+                        <th scope="col" class="govuk-table__header">Error</th>
+                    </tr>
+                </thead>
+                <tbody class="govuk-table__body">
+                    {% for item in validationErrors %}
+                        <tr class="govuk-table__row" data-cy="validation-errors-table-row">
+                            <td class="govuk-table__cell">
+                                {% if item.exporter !== null %}
+                                    {{ item.exporter }}
+                                {% else %}
+                                    -
+                                {% endif %}
+                            </td>
+                            <td class="govuk-table__cell">
+                                {% if item.row !== null %}
+                                    {{ item.row }}
+                                {% else %}
+                                    -
+                                {% endif %}
+                            </td>
+                            <td class="govuk-table__cell">
+                                {% if item.column !== null %}
+                                    {{ item.column }}
+                                {% else %}
+                                    -
+                                {% endif %}
+                            </td>
+                            <td class="govuk-table__cell">{{ item.value }}</td>
+                            <td class="govuk-table__cell">
+                                <span class="govuk-error-message govuk-!-margin-bottom-0">{{ item.errorMessage }}</span>
+                            </td>
+                        </tr>
+                    {% endfor %}
+                </tbody>
+            </table>
+        </div>
+    {% endif %}
+    <form method="POST" data-cy="form" enctype="multipart/form-data">
+        <div class="govuk-form-group">
+            <label class="govuk-label govuk-label--m" for="utilisation-report-file-upload">
+                Upload the report again
+            </label>
+            <div id="utilisation-report-file-upload-hint" class="govuk-hint">
+                You can upload the corrected report now or come back later and upload it
+            </div>
+            <input
+                class="govuk-file-upload govuk-!-width-one-half"
+                id="utilisation-report-file-upload"
+                name="utilisation-report-file-upload"
+                data-cy="utilisation-report-file-upload"
+                data-tag="govuk-file-upload"
+                type="file"
+                accept=".xls, .xlsx, .csv"/>
+        </div>
+        <div class="govuk-button-group">
+            <input
+                formaction="?_csrf={{ csrfToken }}"
+                type="submit"
+                class="govuk-button govuk-!-margin-right-1"
+                data-module="govuk-button"
+                value="Continue"
```

Comment:
It'd be worth checking with a designer if something along the lines of "Upload" or "Upload and continue" would be better text for the button (here and on the previous screen)

Reply:
Just following designs here but yes we definitely need a content designer for these screens to come review them